pkg/koordlet/resmanager/cpu_suppress.go

```diff
@@ -434,7 +437,12 @@ func (r *CPUSuppress) adjustByCfsQuota(cpuQuantity *resource.Quantity, node *cor
 		newBeQuota = *currentBeQuota + int64(beMaxIncreaseCPUQuota)
 	}
 
-	updater, err := resourceexecutor.DefaultCgroupUpdaterFactory.New(system.CPUCFSQuotaName, beCgroupPath, strconv.FormatInt(newBeQuota, 10))
+	auditdata := &audit.Event{
+		Type:    "node",
```

Comment:
should be defined in the resource executor package (e.g. `Type: resourceexecutor.NodeEvent`)